EnglishTranslation-master/ETC.tsv

```diff
@@ -17443,14 +17443,14 @@ ETC_20151224_017444	This isn't a strong monster, but tears could be dropping fro
 ETC_20151224_017445	Poisoned Kepa
 ETC_20151224_017446	It has a poison like the Chinency. However, its poison has a lower effect than the Chinency.
 ETC_20151224_017447	It is a demon form that the souls touched with Evil. The main different is that It has been corrupted to mix demon's soul not human's soul became evil.
-ETC_20151224_017448	Goddess Austeja has been hard to look over the mission, after that, these monsters have been appeared beyond the Goddess's power
+ETC_20151224_017448	Goddess Austeja has been hard to look over the mission, after that, these monsters have been appeared beyond the goddess's power
 ETC_20151224_017449	Dionia gained the strong spell by Demon Master Bramble. So, it seems originaly transformed to monsters from plants, but it maybe think belong to demons.
 ETC_20151224_017450	Dionia is known for his wisdom is week, but he tends to show an affection for his spore only. It's quite strange situation considering the Plant-types.
 ETC_20151224_017451	Almost Golem in the world originally crafted for good purposes, but now, They have been working caused by the evil spirit.
 ETC_20151224_017452	Ellaganos is created with the strong power and form, after the unique undead which mixed with the dead and stone used at the grave over many years.
 ETC_20151224_017453	There are famous comments by someone, this monster is hard to eat and to protect.
 ETC_20151224_017454	Mirtis is considered a low-ranked Demon Lord of all the others. The reason why is probably because she failed many missions that were given to her from Giltine. On her last mission, she was sealed by a human. Since Medzio Diena, she has now the freedom to catch up on her failed mission.
-ETC_20151224_017455	There was something that Damon Master ordered the fatigue duty by making his avatar and dividing his power. This is the same case. In addition, Sorcerer gain advantage by using magics to use like demon's feature.
+ETC_20151224_017455	There was something that a Demon Lord ordered the fatigue duty by making his avatar and dividing his power. This is the same case. In addition, Sorcerer gain advantage by using magics to use like demon's feature.
```

Comment:
would you consider reworking or explaining this line? i'm a bit confused by what they're saying due to the wording so i'm stumped for suggestions too ; 3;